aws-crt/spec/event_loop_group_spec.rb

```diff
@@ -1,8 +1,19 @@
 require 'spec_helper'
+require 'weakref'
 
-describe Aws::Crt::IO do
-  it 'has a test' do
+describe Aws::Crt::IO::EventLoopGroup do
+  it 'lives and dies' do
+    # create
     elg = Aws::Crt::IO::EventLoopGroup.new
-    elg.destroy
+    expect(elg).to_not be_nil
+    weakref = WeakRef.new(elg)
+    expect(weakref.weakref_alive?).to be_truthy
```

Comment:
Typically you'd just do `expect().to be true` or `.to be false`

Reply:
weakref_alive? returns true or nil 😐
so I can do `to be true` but gotta keep `to be_falsey`